pages/vi/vi-planetapps.md

```diff
@@ -84,12 +84,12 @@ Take the course, and the last question will ask you to specify any problems or i
   ![Submissions button](images/vi-planet-submissions-button.png)
   ![View Submissions](images/vi-planet-submissions.png)
 
-6. When you provide feedback, try to be as specific as possible, and include screenshots when necessary
+6. When you provide feedback, try to be as specific as possible, and include screenshots when necessary, through your preferred method (Gitter chat is always a great fallback if you are not comfortable with uploading and linking screenshots).
```

Comment:
@johnmdrake1 can you explain this statement? Are you trying to say you should upload screenshot or saying not to use gitter if you are not familiar with that feature?

Reply:
When I was taking the planet course, there was nowhere I saw in the free response feedback question to upload a photo or screenshot, I checked the toolbar at the top of the text entry field for an attachment button and didn't see one unless I missed it. I personally uploaded my screenshots to Imgur, and made a private Imgur link to my album which I put in my answer to question 12. Not everyone may know how to use screenshot sharing websites such as Imgur, so I was saying use Gitter if you don't know another way. If there is a preferred image hosting site OLE would like Virtual Interns to use for that question that should go there instead.

Reply:
@Immrssa

Reply:
@leomaxi comes up, if that's your tag my apologies

Reply:
@lmmrssa

Reply:
Thank you @Liew211

Reply:
@johnmdrake1 if question has asked to add screenshot then that question needs to be changed. As we currently does not support Image on answer that question might have been migrated from old application. Will make change on course exam itself.

Reply:
@lmmrssa

<img width="1238" alt="Screen Shot 2019-07-10 at 1 19 08 AM" src="https://user-images.githubusercontent.com/32374869/60945321-da2f7d80-a2b0-11e9-9444-445afbcc6e45.png">

Is the Planet course question that asks for screenshots, which I agree are helpful.

The original content of the wiki page also says `When you provide feedback, try to be as specific as possible, and include screenshots when necessary` , so with my additions to that part, I was simply trying to tell the Aspiring Interns how they could upload screenshots for the last course question, since there is no upload screenshot button on planet. Would you like me to change anything, such as suggesting the person to upload to Imgur and post a link in question 12 instead? Or would you like me to take out the part of the wiki page that says to include screenshots. I am a little confused about what you are saying, but please let me know if anything needs to be changed before this pull request is approved regarding screenshots.

Reply:
@lmmrssa How would you like them to upload screenshots in question 12

Reply:
@lmmrssa

For example, I answered course question 12 by uploading my screenshots to Imgur and posting them here https://imgur.com/a/Sf8umdm because that was the easiest way.

Reply:
I had the same problem with uploading images inside Planet for Course - Virtual Intern on the last question: I can not drop or paste or inserte my images like Gitter, so I use feedback in Planet instead. 
![image](https://user-images.githubusercontent.com/33206707/61013706-9ed89180-a338-11e9-991d-6daf1971878b.png)

But I think it is necessary for "step 4" to be clear on how to do the image uploading inside the Planet or there another way to do it.

Reply:
@lillyxxx completely agreed. Thoughts @lmmrssa So I can get this issue and pull request sorted out?

Reply:
@johnmdrake1 @lillyxxx as mentioned planet does not have way to upload image on exam. If question is confusing we can change the question which is not part of this repository.

Reply:
Ok, in my opinion that is what should be done. On my end, should I take out my addition about how to upload images on the exam, and the Planet team can change the question in the appropriate repository? @lmmrssa

Reply:
@lmmrssa is this better?

Reply:
(see most recent commit)